include/odp/api/spec/atomic.h

```diff
@@ -176,6 +177,20 @@ void odp_atomic_dec_u32(odp_atomic_u32_t *atom);
  */
 void odp_atomic_max_u32(odp_atomic_u32_t *atom, uint32_t new_max);
 
+/**
+ * Fetch and update maximum value of atomic uint32 variable
```

Comment:
The description does not say that the whole operation (fetching + updating) is atomic, even though it is implied by the name of the function. This is actually the case in the existing atomic functions too. Maybe the header could have a generic comment somewhere, stating that all the atomic functions perform their operation atomically unless otherwise noted.

Reply:
Comment added in V3.